### ISSUE TITLE
Add support for `text-decoration-skip-ink`

### ISSUE THREAD

**Issue by XhmikosR** [w3c/css-validator#222]
It seems `text-decoration-skip-ink` is in Working Draft phase.

Could you add it to the known properties?

/CC @sideshowbarker 

Thanks!

**Comment by ylafon** [comment]
Considering it is a FPWD, it is unlikely that it already has wide implementation support, so it will not be high on the list, especially as it may change wildly.

**Comment by sideshowbarker** [comment]
https://wpt.fyi/results/css/css-text-decor/text-decoration-skip-ink.html?label=master&product=chrome%5Bexperimental%5D&product=edge&product=firefox%5Bexperimental%5D&product=safari%5Bexperimental%5D&aligned seems to indicate `text-decoration-skip-ink` is not supported in Firefox, Safari, or Edge. It’s only supported Chrome so far

**Comment by sideshowbarker** [comment]
See also https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-skip, which also indicates lack of cross-browser support

**Comment by XhmikosR** [comment]
Yeah, that's correct. We can wait and see how this goes but as you know, with Microsoft's decision to switch Edge to a Chromium-based engine, you know.

But I agree with you, let's wait.

**Comment by XhmikosR** [comment]
It seems Firefox added support for this in the upcoming v70.

**Comment by MichaIng** [comment]
Just giving this a bump, support seems to be now present on most relevant browsers: https://developer.mozilla.org/de/docs/Web/CSS/text-decoration-skip-ink
